pep-9999.rst

```diff
@@ -0,0 +1,316 @@
+checking for glibc. If the platform has glibc 2.17 or newer, it is
+assumed to be compatible unless the ``_manylinux`` module says
+otherwise.
+
+Specifically, the algorithm we propose is::
+
+    def is_manylinux2014_compatible():
+        # Only Linux, and only supported architectures
+        from distutils.util import get_platform
+
+        if get_platform() not in [
+            "linux-x86_64",
+            "linux-i686",
+            "linux-aarch64",
+            "linux-armhfp",
+            "linux-ppc64",
+            "linux-ppc64le",
+            "linux-s390x",
+        ]:
+            return False
+
+        # Check for presence of _manylinux module
+        try:
+            import _manylinux
+
+            return bool(_manylinux.manylinux2014_compatible)
+        except (ImportError, AttributeError):
+            # Fall through to heuristic check below
+            pass
+
+        # Check glibc version. CentOS 7 uses glibc 2.17.
+        # PEP 513 contains an implementation of this function.
+        return have_compatible_glibc(2, 17)
+
+Backwards compatibility with ``manylinux2010`` wheels
+=====================================================
+
+As explained in :pep:`513`, the specified symbol versions for
+``manylinux1`` allowed libraries constitute an *upper bound*.  The
+same is true for the symbol versions defined for ``manylinux2014`` in
+this PEP.  As a result, ``manylinux1`` and ``manylinux2010`` wheels
+are considered ``manylinux2014`` wheels.  A ``pip`` that recognizes
+the ``manylinux2014`` platform tag will thus install ``manylinux2010``
+wheels for ``manylinux2014`` platforms -- even when explicitly set --
+when no ``manylinux2014`` wheels are available.
+
+PyPI Support
+============
+
+PyPI should permit wheels containing the ``manylinux2014`` platform
+tag to be uploaded in the same way that it permits ``manylinux2010``.
+It should not attempt to verify the compatibility of ``manylinux2014``
+wheels.
+
+References
+==========
+
+.. [1] CentOS Product Specifications
+   (https://wiki.centos.org/About/Product)
+.. [2] PEP 425 -- Compatibility Tags for Built Distributions
+   (https://www.python.org/dev/peps/pep-0425/)
+.. [3] Tracking issue for manylinux2010 rollout
+   (https://github.com/pypa/manylinux/issues/179)
+.. [4] Red Hat Universal Base Image 7
+   (https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/ubi7)
+   (https://wiki.centos.org/SpecialInterestGroup/AltArch)
+.. [5] The CentOS Alternative Architecture Special Interest Group
+   (https://wiki.centos.org/SpecialInterestGroup/AltArch)
+.. [6] PEP 3149
+   https://www.python.org/dev/peps/pep-3149/
+.. [7] SOABI support for Python 2.X and PyPy
+   https://github.com/pypa/pip/pull/3075
+.. [8] manylinux2014 Docker images
+   (https://quay.io/repository/pypa/manylinux2014_x86_64)
+.. [9] auditwheel
+   (https://github.com/pypa/auditwheel/)
+
+Copyright
+=========
+
+This document has been placed into the public domain.
```

Comment:
Hello @di,

The requirements at [PEP 1](https://www.python.org/dev/peps/pep-0001/#what-belongs-in-a-successful-pep) was changed recently in #1117, so can you please update `Copyright` section to make this PEP meet them?
> 13. Copyright/license -- Each new PEP must be placed under a dual license of public domain and CC0-1.0-Universal [8] (see this PEP for an example).

```suggestion
This document is placed in the public domain or under the CC0-1.0-Universal license, whichever is more permissive.
```